### ISSUE TITLE
Integrate learnings from SRR XNAT container service dockerisation into ansible

### ISSUE THREAD

**Issue by ruaridhg** [UCL-MIRSG/ansible-collection-infra#153]
### Short Description

There were certain challenges with running SRR imaging processing where variables such as limits for docker container were implemented with the SRR repo and could therefore be integrated in the main ansible code.

**Comment by drmatthews** [comment]
@ruaridhg - is there anything else to be done to the `docker` or `xnat_container_service` Ansible roles resulting from what was done in the SRR project? If there is anything, can you please add issues to the collection repo.

**Comment by ruaridhg** [comment]
> [@ruaridhg](https://github.com/ruaridhg) - is there anything else to be done to the `docker` or `xnat_container_service` Ansible roles resulting from what was done in the SRR project? If there is anything, can you please add issues to the collection repo.

No, I think the changes were made with docker swarm which was the main thing.
